kokoro/linux/python/resolve_docker_tag.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/python
+
+"""Retrieve an image's information."""
+
+import re
+import sys
+
+def _parse_image(name):
+    m = re.match(r'^(?:(?P<host>[\w.-^_]+(?:[:][\d]+))/)?(?P<name>[\w.-]+/[\w.-]+)(?:[:](?P<tag>[\w.-]+))?$',
```

Comment:
can you add a comment with a few examples of strings you expect this to match against